foca/models/config.py

```diff
@@ -637,8 +637,6 @@ class AuthConfig(FOCABaseConfig):
             are rejected. If `None`, audience validation is disabled.
         claim_identity: The JWT claim used to identify the sender.
         claim_issuer: The JWT claim used to identify the issuer.
-        claim_key_id: The JWT claim used to identify the JWK used when the JWT
-            was issued.
```

Comment:
Indeed I removed some `AuthConfig` model properties during the last refactoring so that's why you were getting these errors. I had forgotten to remove one of them from one of the docstrings though, so here I removed that.